_includes/templates/primary-nav.html

```diff
@@ -1,4 +1,5 @@
 {% import "nav-items-toplevel.html" as vars with context %}
+<div class="sliding-nav_nav">
```

Comment:
Any way we can reduce the amount of wrapping elems? Seems like there's a lot of nesting that may not be necessary.